v-next/hardhat/console.sol

```diff
@@ -7,8 +7,7 @@ library console {
 
     function _sendLogPayloadImplementation(bytes memory payload) internal view {
         address consoleAddress = CONSOLE_ADDRESS;
-        /// @solidity memory-safe-assembly
-        assembly {
+        assembly ('memory-safe') {
```

Comment:
I think this changes make the libraries compatible with less versions of solc.

I guess instead of doing that, we can filter out these warnings.